### PR TITLE
Button Component

### PR DESCRIPTION
**Pull request by Alee-star**
Button Component

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,8 +1,13 @@
 import React from "react";
 import "./App.css";
+import Heading from "./components/Header/Heading";
```
Comment (vonnue-nidheesh) [inline src/App.js]: Sort imports

File: src/components/Button.jsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+const Button = ({ label, onClick, isDisabled = false, className }) => {
```
Comment (vonnue-nidheesh) [inline src/components/Button.jsx]: Try to write optional props at last
`const Button = ({ label, onClick, className, isDisabled = false }) => {`

File: src/components/Button.jsx [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+  return (
+    <button className={className} onClick={onClick} disabled={isDisabled}>
+      {label}
+    </button>
```
Comment (vonnue-nidheesh) [inline src/components/Button.jsx:7]: Missing button style CSS file. Why the button component is not styled? Now it shows the basic state right?

Comment (Alee-star) [reply src/components/Button.jsx:7]: The button is used in the Navbar section and has been styled using its class name. Is a button style is needed ?

**Review by vonnue-nidheesh — APPROVED** [review]

**Pull request merged**